src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -1985,6 +2152,149 @@ await using (var context = InMemoryContentDbContext(contextId))
             }
         }
 
+        [Fact]
+        public async Task GetPublicationTree_DataTables_ReleaseHasData()
```

Comment:
Is this already covered by `GetPublicationTree_DataTables_LatestDataNotSuperseded`? We might want to adjust the name to make it more readable/obvious.

These new assertions should probably be in there instead.